testsuite/features/secondary/min_salt_migration.feature

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+# This feature depends on 
+# - minkvm_guests.feature which has already configured the KVM Minion to be a virtualization host.
+# - srv_sync_channels.feature which syncs SLES 15 SP4 and the client tools.
+# We also test 'Bootstrapping using the command line' in this feature
+
+@scope_salt
+@scope_salt_ssh
+@scope_virtualization
+@virthost_kvm
+Feature: Migrate Salt to bundled Salt on a nested Minion VM
+
+  Scenario: Log in as admin user in the Salt migration context
+    Given I am authorized for the "Admin" section
+
+  Scenario: Prepare a SLES KVM test VM and list it in the Salt migration context
+    Given I am on the Systems overview page of this "kvm_server"
+    When I delete default virtual network on "kvm_server"
+    And I create salt-sles virtual network on "kvm_server"
+    And I delete default virtual storage pool on "kvm_server"
+    And I create salt-sles virtual storage pool on "kvm_server"
+    And I create a sles virtual machine named "sles-salt-migration" on "kvm_server"
+    And I follow "Virtualization" in the content area
+    And I wait until I see "sles-salt-migration" text
+
+  Scenario: Start the SLES 15 KVM VM in the Salt migration context
+    Given I follow "Virtualization" in the content area
+    When I click on "Start" in row "sles-salt-migration"
+    Then I should see "sles-salt-migration" virtual machine running on "kvm_server"
+    # We need a small delay to give cloud init some time to do its magic
+    And I wait for "30" seconds
```

Comment:
This still under the "Preparation for the test" isn't it?

Reply:
Yes. The actual test starts with `Scenario: Do some basic testing on the SLES 15 KVM VM without Salt bundle` after the Minion got onboarded without the Salt bundle.